valhalla/meili/measurement.h

```diff
@@ -15,9 +15,10 @@ class Measurement {
   Measurement(const midgard::PointLL& lnglat,
               float gps_accuracy,
               float search_radius,
-              double epoch_time = -1.f)
+              double epoch_time = -1.f,
+              bool is_break_point = false)
       : lnglat_(lnglat), gps_accuracy_(gps_accuracy), search_radius_(search_radius),
-        epoch_time_(epoch_time) {
+        epoch_time_(epoch_time), is_break_point_(is_break_point) {
```

Comment:
Should this actually be `location_type{Location::StopType::BREAK}` as opposed to a very specific boolean?